src/orange-jam.md

```diff
@@ -0,0 +1,28 @@
+# Orange jam
+
+This jam is a traditional topping for the Polish [mazurek](mazurek.html), but
+can be used as a generic sweet spread.
+
+## Ingredients
+
+* 4 medium sized oranges
+* 1 lemon
+* 1 1/2 cup sugar
+
+## Directions
+
+1. Zest all fruit.
+2. Peel of the albedo.
```

Comment:
```suggestion
2. Peel off the albedo (the white part under the skin of some fruit).
```

:O I didn't know there was a name for the white stuff. Maybe a short description would help others who also didn't know.

Reply:
Thanks for the typo catch. Added a short description.